setup.cfg

```diff
@@ -44,6 +44,7 @@ install_requires =
     pandas >= 1.3.0, <1.4.2
     scikit-learn >= 0.22
     python-dateutil >= 2.8.1
+    scipy >= 1.4.0
```

Comment:
`scipy < 1.4.0` install fails with `scikit-learn == 0.22` to `0.24`. Add this as a requirement to prevent install failures